src/DI/ClassAliasMap.php

```diff
@@ -3,6 +3,6 @@
 declare(strict_types = 1);
 
 return [
-	\Kdyby\DoctrineCache\NotImplementedException::class 	=> \Kdyby\DoctrineCache\Exception\NotImplementedException::class,
-	\Kdyby\DoctrineCache\Exception::class 					=> \Kdyby\DoctrineCache\Exception\Exception::class,
+	\Kdyby\DoctrineCache\Exception\NotImplementedException::class => \Kdyby\DoctrineCache\Exception\NotImplementedException::class,
```

Comment:
This should not be here. You are aliasing same class name. This is here just to provide backwards compatibility for renamed namespaced class `\Kdyby\DoctrineCache\NotImplementedException`